Resources/views/Security/login_content.html.twig

```diff
@@ -9,14 +9,11 @@
         <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
     {% endif %}
 
-    <label for="username">{{ 'security.login.username'|trans }}</label>
-    <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" />
+    {{ form_row(form._username, {'full_name': '_username', 'label': 'security.login.username'|trans}) }}
```

Comment:
Can you put the attributes (full_name, label) to the PHP part? All other template do the same.